test/libsolidity/semanticTests/revertStrings/function_entry_checks_v1.sol

```diff
@@ -4,7 +4,7 @@ contract C {
 }
 // ====
 // EVMVersion: >=byzantium
-// compileViaYul: false
+// compileViaYul: also
```

Comment:
Also in this file the `v1` on top is useless now, since the revert string was implemented.